src/helpers/splitLastWord.js

```diff
@@ -0,0 +1,11 @@
+export const removeLastWord = (text) => {
```

Comment:
I don't feel like the filename represents these two helpers. I'd just maybe just call it `helpers/words.js` or something

Reply:
f64c08e